awesomplete.js

```diff
@@ -45,10 +46,17 @@ var _ = function (input, o) {
 		around: input
 	});
 
-	this.ul = $.create("ul", {
-		hidden: "",
-		inside: this.container
-	});
+	this.ul = this.dropdown ?
+		$.set(this.dropdown, {
+			hidden: '',
+			className: 'awesomplete-dropdown',
+			inside: this.container
+		}) :
+		$.create('ul', {
+			hidden: '',
+			className: 'awesomplete-dropdown',
+			inside: this.container
+		});
```

Comment:
I would rather not duplicate everything here. The options object can be a variable that is shared by both function calls.